src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionTests_WithElement_Constructor.cs

```diff
@@ -2002,9 +2049,6 @@ static void Goo<T>(MyList<T> list) { }
             // (1,1): hidden CS8019: Unnecessary using directive.
             // using System;
             Diagnostic(ErrorCode.HDN_UnusedUsingDirective, "using System;").WithLocation(1, 1),
-            // (15,13): warning CS0219: The variable 'i' is assigned but its value is never used
```

Comment:
The related prototype comment should be deleted also, but, GitHub suggestion range is not broad enough to let me suggest doing that.

Reply:
cc91315ba12